app/views/users/index.html.erb

```diff
@@ -0,0 +1,12 @@
+<div id="users-container">
+    <% @users.each do |user| %>
+      <div class="user">
+        <img class="user-img" src="<%= user.photo %>" alt="User photo">
+        <div class="name-posts-count">
+            <h2 class="user-name"><%= link_to user.name, user_path(user) %></h2>
+
+            <div class="user-posts-count"><p>Number of posts: <%= user.post_counter %></p></div>
+        </div>
+      </div>
+    <% end %>
+</div>
```

Comment:
- [ ] You're required to have the same form for posts in the view. Kindly do just that.

### Project Requirement.
![view](https://github.com/sheezu900/blog-rails/assets/26597432/f15366e6-c8f2-4c14-b9da-a34b641118e3)